shell/app/org-home/stores/org.tsx

```diff
@@ -80,6 +80,10 @@ const org = createStore({
       } else {
         const currentOrg = resOrg || {};
         const orgId = currentOrg.id;
+        // user doesn't joined the public org, go to workBench
+        if (resOrg.isPublic && !orgs?.list?.find((x: ORG.IOrg) => x.name === currentOrg.name)) {
```

Comment:
actually this `x` type can be inferred by ts automatically. no need `: ORG.IORG`